nodejs/dist/package.json

```diff
@@ -2,10 +2,14 @@
   "name": "uws",
   "version": "0.3.1",
   "scripts": {
-    "build": ""
+    "preinstall": "node-gyp rebuild && exit 0; echo 'Falling back to precompiled binaries';"
```

Comment:
Maybe some kind of check with `which node-gyp` or something like that could work? How complex can this preinstall script be?